examples/laser_ionization/analysis_laser_ionization.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env python3
+
+#
+# This file is part of HiPACE++.
+#
+# Authors: EyaDammak
+
+import argparse
+import numpy as np
+import math
+from openpmd_viewer import OpenPMDTimeSeries
+import statistics
+
+parser = argparse.ArgumentParser(
+    description='Script to analyze the equality of two simulations')
+parser.add_argument('--first',
+                    dest='first',
+                    required=True,
+                    help='Path to the directory containing output files')
+parser.add_argument('--second',
+                    dest='second',
+                    required=True,
+                    help='Path to the directory containing output files')
+args = parser.parse_args()
+
+ts_linear = OpenPMDTimeSeries(args.first)
+ts_circular = OpenPMDTimeSeries(args.second)
+
+a0_linear = 0.00885126
+a0_circular = 0.00787934
+
+nc = 1.75e27
+n0 = nc / 10000
+
+qe = 1.602176634e-19
+
+iteration = 0
+
+rho_elec_linear, _ = ts_linear.get_field(field='rho_elec', coord='z', iteration=iteration, plot=False)
```

Comment:
I think this is the default value, so no need to specify it.
```suggestion
rho_elec_linear, _ = ts_linear.get_field(field='rho_elec', coord='z', iteration=iteration)
```